docs/devops/getting-started.md

```diff
@@ -22,18 +22,28 @@ image:
 ## Best DevOps practices at SAP ##
 
 Continuous delivery is a method to develop software with short feedback cycles.
-It is applicable to projects both for SAP Cloud Platform and SAP on-premise platforms.
-SAP implements tooling for continuous delivery in project "Piper".
-The goal of project "Piper" is to substantially ease setting up continuous delivery in your project using SAP technologies.
+It applies to projects both for SAP Cloud Platform and SAP on-premise platforms.
+SAP implements tooling for continuous delivery in the project "Piper".
```

Comment:
I'm not sure what a native speaker would say, but I'd not prefix "project "Piper"" with "the".

Reply:
Well, me neither, but definitive article grammatically correct and seems to be favored by my grammar checking tool. Otherwise - to me, it's low importance as doesn't change the meaning.